kmath-tensors/src/commonMain/kotlin/space/kscience/kmath/tensors/api/StatisticTensorAlgebra.kt

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2018-2021 KMath contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package space.kscience.kmath.tensors.api
+
+import space.kscience.kmath.tensors.core.DoubleTensor
+
+/**
+ * Common algebra with statistics methods. Operates on [Tensor].
+ */
+
+public interface StatisticTensorAlgebra<T>: TensorAlgebra<T> {
```

Comment:
Better to call it TensorStatistics. And I am not sure it should inherit TensorAlgebra. It could be a separate interface so we could add statistics to existing tensor algebras which does not support statistics like this:
```kotlin
TensorAlgebra{
  statistics{
    a.min()
  }
}
```